### ISSUE TITLE
New Organization?

### ISSUE THREAD

**Issue by Archmonger** [Archmonger/django-dbbackup#483]
Things have been moving slowly on https://github.com/jazzband/help/issues/196, which has made it difficult/impossible to fix our broken CI. This subsequently has caused us to be hesitant of [significant overhauls](https://github.com/jazzband/django-dbbackup/issues/423) to django-dbbackup due to the inevitable need for updating the CI. Additionally, the jazzband [user registration workflow](https://github.com/jazzband/help/issues/288) has been broken for many users, making it adding new maintainers a challenge.

I've started the [request to transfer out](https://github.com/jazzband/help/issues/336). But, we should discuss what potential orgs this repo can be moved to for increased sustainability.

The immediate options that come to mind are
- [django-dbbackup](https://github.com/django-dbbackup/)
- [revsys](https://github.com/revsys)
- [encode](https://github.com/encode)
- [django-commons](https://github.com/django-commons)

This issue is to open discuss what orgs we have in mind for transfer. Feel free to vote on the existing options, or to suggest new options.

cc: @johnthagen @benjaoming @pkkid @ZuluPro @jonathan-s

**Comment by johnthagen** [comment]
The old PyPI releases point to https://github.com/django-dbbackup/django-dbbackup

Perhaps easiest to simply go back to where we were before jazzband?

**Comment by pkkid** [comment]
Anything would be fine on my end, if there is anything I can do to help, let me know.

**Comment by Archmonger** [comment]
Yup, going back to the old org would be the simplest solution. We did originally moved orgs to add more maintainers, however, we don't want a re-org to negatively impacts maintainability.

Jazzband's concept and registration structure is nice, but unfortunately the org has been running on fumes for seemingly ~4 years.

To put it more bluntly, are there any python/django related orgs that are worth considering?

`revsys` is a dedicated group but seems like they limit who joins the org. `encode` might be closer to what we're looking for, but I don't know what it's like to operate under that org.

---

Pinging the guys in `encode` I've seen around while working PRs: @auvipy @kludex @tomchristie

Could you give some opinions on this? Are there any challenges of working within the `encode` org, and do you think this repo might fit within within the org?

There also doesn't seem to be guidelines on what repos can join, and what the org structure is like.

**Comment by Kludex** [comment]
I know nothing about Django. @adamchainz may be a better ping than me. 👀

**Comment by adamchainz** [comment]
I don't really do anything with jazzband or any org, I mostly maintain a bunch of repos myself.

**Comment by benjaoming** [comment]
@Archmonger I think your concerns around Jazzband has been around having freedom to setup your own CI stuff, onboard new maintainers etc. To guarantee that, moving forwards with django-dbbackup's own organization seems like a good fit :+1:

**Comment by pkkid** [comment]
I organization I had under my account was unused. So just cleaning things up.

**Comment by Archmonger** [comment]
[Django Commons](https://github.com/django-commons/) seems to be a new potential option for this repo.
